templates/search-result.tmpl

```diff
@@ -136,12 +136,24 @@
     
     <div class="navigation">
       
-      {if $previous >= 0}
+    {if $previous >= 0}
 	<span class="txtPrev">
-	  <a href="#" onclick="javascript:jumpto({$previous}); return false;">&laquo; Previous</a>
+	  <a href="#" onclick="javascript:jumpto({$previous}); return false;">&laquo;&nbsp;Previous</a>
 	</span>
 	{/if}
-
+    {for $f in range(max(0, floor($previous/$limit) - 4), 
```

Comment:
-4 and +5 in the next line are related to the step, no?  Notice that the step is configurable (see method `get-next` in `web-service.lisp`, default is 10).  We should probably pass the step as a parameter and use the parameter as opposed to hardcoding numbers in the template file.